py_zipkin/zipkin.py

```diff
@@ -126,6 +127,9 @@ def __init__(
         :param transport_handler: Callback function that takes a message parameter
             and handles logging it
         :type transport_handler: function
+        :param max_span_portion_size: Spans in a trace are sent in batches,
```

Comment:
Yeah I like the idea of having "batch" somewhere in the name, considering the batch sender class is called `ZipkinBatchSender`